packages/widget/src/components/transfer/fungible/fungible-token-transfer.ts

```diff
@@ -73,6 +75,28 @@ export class FungibleTokenTransfer extends BaseComponent {
     }
   };
 
+  renderAmountOnDestination(): HTMLTemplateResult | null {
+    if (
+      this.transferController.selectedResource &&
+      this.transferController.pendingTransferTransaction !== undefined
+    ) {
+      const { decimals, symbol } = this.transferController.selectedResource;
+      return html`
+        <div class="amountOnDestination">
+          <span> Amount to receive: </span>
+          <span>
+            ${utils.formatUnits(
```

Comment:
why not use internal `tokenBalanceToNumber`?

Reply:
let me check it

Reply:
works